tests/unit/MotionAnimatorTests.swift

```diff
@@ -49,6 +49,8 @@ class MotionAnimatorTests: XCTestCase {
 
   func testAnimatorOnlyAddsAnimationsForKeyPath() {
```

Comment:
Naming? `testAnimatorOnlyUsesSingleNonAdditiveAnimationForKeyPath`

Also, a comlementary test to confirm that 2 `additive = true` animations will both be added?

Reply:
Fixed naming.

And unfortunately (again, because of nil keys), we can't test the number of animations added to a layer when the animator is using additive animations :(